firestore/src/main/aggregate_query_main.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef FIREBASE_FIRESTORE_SRC_MAIN_AGGREGATE_QUERY_MAIN_H_
+#define FIREBASE_FIRESTORE_SRC_MAIN_AGGREGATE_QUERY_MAIN_H_
+
+#include "Firestore/core/src/api/aggregate_query.h"
+#include "firestore/src/include/firebase/firestore/aggregate_query.h"
+#include "firestore/src/main/firestore_main.h"
+
+#if defined(__ANDROID__)
+#error "This header should not be used on Android."
+#endif
+
+namespace firebase {
+namespace firestore {
+
+class AggregateQueryInternal {
+ public:
+  explicit AggregateQueryInternal(api::AggregateQuery&& aggregate_query);
+  ~AggregateQueryInternal() = default;
```

Comment:
Is there a reason to explicitly declare the destructor? I think this can just be omitted.

Reply:
No. Removed.